aws_iot_update_shadow_block.py

```diff
@@ -0,0 +1,55 @@
+import json
+from copy import deepcopy
+from nio.properties import VersionProperty, StringProperty, Property
+from nio import TerminatorBlock
+from AWSIoTPythonSDK.MQTTLib import AWSIoTMQTTShadowClient
+from AWSIoTPythonSDK.exception import AWSIoTExceptions
+from .aws_iot_mqtt_base_block import AWSIoTMQTTBase
+
+
+class AWSIoTUpdateShadow(AWSIoTMQTTBase, TerminatorBlock):
+    """Update a device's shadow document with a reported state."""
+
+    version = VersionProperty('1.0.0')
+    thing_name = StringProperty(title='Thing Name')
+    reported_state = Property(
+        title='Reported State', default='{{ $.to_dict() }}')
+
+    def __init__(self):
+        super().__init__()
+        self.client = deepcopy(AWSIoTMQTTShadowClient)
```

Comment:
What's this `deepcopy` about? Seems bizarre

Reply:
I was simply following the pattern in the base's `__init__`
https://github.com/nio-blocks/AWS_IoT/blob/shadow_update/aws_iot_mqtt_base_block.py#L40

Looking at history now to try and figure out why.

Reply:
Ok yeah, that seems very strange. See if you can track it down and maybe what happens if we get rid of it. It's possible it was there to "test" something and never got removed. Can't imagine why we'd want to deepcopy a 3rd party library's class